lib/client.c

```diff
@@ -525,6 +525,7 @@ _snowflake_check_connection_parameters(SF_CONNECT *sf) {
     log_debug("protocol: %s", sf->protocol);
     log_debug("autocommit: %s", sf->autocommit ? "true": "false");
     log_debug("insecure_mode: %s", sf->insecure_mode ? "true" : "false");
+    log_debug("ocsp_fail_open: %s", sf->ocsp_fail_open ? "true" : "false");
```

Comment:
let's keep the default to false to not cause BCR and we can merge to master

Reply:
done https://github.com/snowflakedb/libsnowflakeclient/pull/740/files#diff-1573e024f58fe545f3b811283ce6ce75f7fa32eec02ef00794877e71941e9cf0R681